observer.go

```diff
@@ -0,0 +1,89 @@
+package zipkintracer
+
+import (
+	opentracing "github.com/opentracing/opentracing-go"
+)
+
+// Observer can be registered with the zipkin to recieve notifications
+// about new Spans.
+// The actual registration depends on the implementation, which might look
+// like the below e.g :
+// observer := myobserver.NewObserver()
+// tracer := zipkin.NewTracer(..., zipkin.WithObserver(observer))
+//
+type Observer interface {
```

Comment:
Please remove the Observer interface here and use the one you created in https://github.com/opentracing-contrib/go-observer. You can pull PR https://github.com/opentracing-contrib/go-observer/pull/2 to fix the missing dependencies to opentracing-go